### DISCUSSION TITLE
Has Anyone tried MESHROOM ?

### DISCUSSION THREAD

**Discussion by o0LINNY0o** [vosen/ZLUDA#149] [Q&A] [answered]
has anyone tried Meshroom ? 
https://github.com/alicevision/Meshroom

I know there is a CL version for AMD ,,, but i went down that rabbit hole , went mad and gave up on it.

**Comment by vosen** [comment]
Some people in #79 tried, unsuccessfully. I'm working on it in #153

**Reply by vosen** [reply, marked as answer]
I'm marking as answered, since support is merged now
